pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -1213,8 +1214,8 @@ func TestNodeOperators(t *testing.T) {
 
 			// Generations are globally unique. We check in our unit tests that they are incremented correctly.
 			expected.Generation = got.info.Generation
-			if !reflect.DeepEqual(got.info, expected) {
-				t.Errorf("Failed to add node into scheduler cache:\n got: %+v \nexpected: %+v", got, expected)
+			if diff := cmp.Diff(expected, got.info, cmp.AllowUnexported(framework.NodeInfo{})); diff != "" {
+				t.Errorf("Failed to add node into scheduler cache (-want, +got):\n%s", diff)
```

Comment:
change message to something like "Unexpected node info from cache....

Reply:
Done. Thanks for review.